python/lsst/summit/utils/utils.py

```diff
@@ -481,3 +485,116 @@ def getSite():
 
     # we have failed
     raise ValueError('Location could not be determined')
+
+
+def getAltAzFromSkyPosition(skyPos, visitInfo):
+    """Get the alt/az from the position on the sky and the time and location
+    of the observation.
+
+    Parameters
+    ----------
+    skyPos : `lsst.geom.SpherePoint`
+        The position on the sky.
+    visitInfo : `lsst.afw.image.VisitInfo`
+        The visit info containing the time of the observation.
+
+    Returns
+    -------
+    alt : `lsst.geom.Angle`
+        The altitude.
+    az : `lsst.geom.Angle`
+        The azimuth.
+
+    Notes
+    -----
+    Currently, as only the AuxTel is on-sky, the location is assumed to be
+    ``AUXTEL_LOCATION``. When these utils are extended to the main telescope,
+    this will need correcting.
+    """
+    skyLocation = SkyCoord(skyPos.getRa().asRadians(), skyPos.getDec().asRadians(), unit=u.rad)
+    altAz = AltAz(obstime=visitInfo.date.toPython(), location=AUXTEL_LOCATION)
```

Comment:
The irony being that `ObservationInfo` has all the information you need, including the Astropy AltAz object. More importantly, `VisitInfo` has the observatory location stored in it so you should not be guessing. `VisitInfo.getObservatory` returns `lsst.afw.coord.Observatory` which you should be able to transform into an astropy location.

Reply:
Good plan. Trivial to add to the code, torture to add to the test suite. Any idea how I'd add the location to the obsInfo/visitInfo I'm crafting by hand? I've tried a few things, but don't even really know where to look to work it out...

Reply:
I tried adding some values for `telescope` to the base header dict but that didn't work.

Reply:
Are you making a VisitInfo from scratch or going via a header and ObservationInfo. The code to go from astropy location to Observatory is here: 

https://github.com/lsst/obs_base/blob/main/python/lsst/obs/base/makeRawVisitInfoViaObsInfo.py#L207-L212

Reply:
Via a "header" and ObservationInfo (where the scare quotes denote it being a hand-made dict rather than an actual FITS header). You can see the code [here](https://github.com/lsst-sitcom/summit_utils/blob/6385fb5fe30719c8c124703a5339175cbb2488ee/tests/test_utils.py#L89)

Reply:
Going between astropy location to Observatory isn't a problem, it's getting the test obsInfo/visitInfo to have it in it that's the problem.

Reply:
What I meant by not knowing where to look is not really understanding what things to shove into the header to make it pop out in the obsInfo.

Reply:
The location comes from the OBSGEO-* FITS headers.

Reply:
Right, but sticking values for those into the baseHeader dict doesn't get them into the output visitInfo, which is what I mean about not getting how to do this 😬